scripts/push-new-nix-store-paths.sh

```diff
@@ -9,4 +9,11 @@ if [ -z "$FLOX_SUBSTITUTER" ]; then
 fi
 
 # Allow pushing to fail.
-cat /tmp/drv-paths | xargs -I{} -r nix copy --extra-experimental-features nix-command --to "$FLOX_SUBSTITUTER" {}^* -vv||:;
+
+# copy the outputs of drv-paths
+# https://www.haskellforall.com/2022/10/how-to-correctly-cache-build-time.html
+
+if [ -f /tmp/drv-paths ]; then
+	cat /tmp/drv-paths | xargs nix-store --query --requisites --include-outputs > /tmp/dependency-paths ||:;
+	cat /tmp/dependency-paths | awk '!seen[$0]++' | xargs nix copy --extra-experimental-features nix-command --to "$FLOX_SUBSTITUTER" ||:;
```

Comment:
question(non-blocking): Why not use `|| true`? First time I've seen this syntax.

Reply:
it was the style used originally when the script was written, so I was just matching it. It does look a bit odd.